src/interfaces/UploadOptions.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Options used when uploading a file
+ *
+ * @see {@link https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload#request-structure-multipart-form-data}
+ */
+export interface UploadOptions {
+  /**
+   * This field accepts three kinds of values:
+   * - binary - You can send the content of the file as binary. This is used when a file is being uploaded from the browser.
+   * - base64 - Base64 encoded string of file content.
+   * - url - URL of the file from where to download the content before uploading.
+   *      Downloading file from URL might take longer, so it is recommended that you pass the binary or base64 content of the file.
+   *      Pass the full URL, for example - https://www.example.com/rest-of-the-image-path.jpg.
+   */
+  file: string | Buffer | File;
```

Comment:
What is `File`?

Reply:
`File` is a Browser Only API to upload files https://developer.mozilla.org/en-US/docs/Web/API/File

It is essentially a subclass of `Blob`